appveyor.yml

```diff
@@ -8,49 +8,32 @@ environment:
     # Avoid long paths on Windows
     STACK_ROOT: "c:\\s"
     STACK_WORK: ".w"
-    CACHE_DIR: "c:\\c"
     WORK_DIR: "c:\\w"
+    CACHE_S3_VERSION: v0.1.1
+    AWS_REGION: us-west-1
```

Comment:
@domenkozar, do we really want to use `us-west-1`?  I think something EU-based like `eu-central-1`, or alternatively `eu-west-1` (for older auth protocol version support, if needed) would make more sense.

Reply:
@deepfire It's better to place the bucket in the region closest to AppVeyor servers, so the cache can be uploaded/downloaded quicker. I don't know if @domenkozar decided on `us-west-1`, experimentally, but considering that AppVeyor is registered in Canada, I'd expect placing S3 bucket in Europe would be slower.

Reply:
Understood, makes sense, thanks!